content/id/docs/concepts/architecture/controller.md

```diff
@@ -0,0 +1,178 @@
+mendekati ke keadaan yang diinginkan, dengan menghidupkan atau mematikan 
+perangkat.
+
+Di Kubernetes, controller adalah kontrol tertutup yang mengawasi keadaan cluster
+{{< glossary_tooltip term_id="cluster" >}} Anda, lalu membuat atau meminta 
+perubahan jika diperlukan. Setiap controller mencoba untuk memindahkan status 
+cluster saat ini mendekati keadaan yang diinginkan.
+
+{{< glossary_definition term_id="controller" length="short">}}
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Pola controller 
+
+Sebuah controller melacak sekurang-kurangnya satu jenis *resources* dari 
+Kubernetes.
+[objects](/docs/concepts/overview/working-with-objects/kubernetes-objects/) ini
+memiliki *spec field* yang merepresentasikan keadaan yang diinginkan. Satu atau 
+lebih controller untuk *resource* tersebut bertanggung jawab untuk membuat 
+keadaan sekarang mendekati keadaan yang diinginkan.
+
+Controller mungkin saja melakukan tindakan itu sendiri; namun secara umum, di 
+Kubernetes, controller akan mengirim pesan ke
+{{< glossary_tooltip text="API server" term_id="kube-apiserver" >}} yang 
+mempunyai efek samping yang bermanfaat. Anda bisa melihat contoh-contoh 
+di bawah ini.
+
+{{< comment >}}
+Beberapa controller bawaan, seperti controller *namespace*, bekerja pada objek
+yang tidak memiliki *spec*. Agar lebih sederhana, halaman ini tidak 
+menjelaskannya secara detail.
+{{< /comment >}}
+
+### Kontrol melalui server API
+
+Controller {{< glossary_tooltip term_id="job" >}} adalah contoh dari controller
+bawaan dari Kubernetes. Controller bawaan tersebut mengelola status melalui
+interaksi dengan server API dari suatu cluster.
+
+Job adalah *resource* dalam Kubernetes yang menjalankan a
+{{< glossary_tooltip term_id="pod" >}}, atau mungkin beberapa Pod sekaligus, 
+untuk melakukan sebuah pekerjaan dan kemudian berhenti.
+
+(Setelah [dijadwalkan](../../../../en/docs/concepts/scheduling/), objek Pod 
+akan menjadi bagian dari keadaan yang diinginkan oleh kubelet).
+
+Ketika controller job melihat tugas baru, maka controller itu memastikan bahwa, 
+di suatu tempat pada cluster Anda, kubelet dalam sekumpulan Node menjalankan 
+Pods dengan jumlah yang benar untuk menyelesaikan pekerjaan. Controller job 
+tidak menjalankan sejumlah Pod atau kontainer apa pun untuk  dirinya sendiri. 
+Namun, controller job mengisyaratkan kepada server API untuk  membuat atau 
+menghapus Pod. Komponen-komponen lain dalam
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}}
+bekerja berdasarkan informasi baru (adakah Pods baru untuk menjadwalkan dan 
+menjalankan pekerjan), dan pada akhirnya pekerjaan itu selesai.
+
+Setelah Anda membuat Job baru, status yang diharapkan adalah bagaimana 
+perkerjaan itu bisa selesai. Controller job membuat status pekerjaan saat ini 
+agar mendekati dengan keadaan yang Anda inginkan: membuat Pod yang melakukan 
+pekerjaan yang Anda inginkan untuk job tersebut, sehingga job hampir 
```

Comment:
job is imported word, should use italic
```suggestion
pekerjaan yang kamu inginkan untuk _job_ tersebut, sehingga _job_ hampir 
```

Reply:
It might be better to write Job (the resource name), without an italic. Alternatively, use a localized term that means “job”.

Reply:
> It might be better to write Job (the resource name), without an italic. Alternatively, use a localized term that means “job”.

I prefer this.